### PR TITLE
Add links to demo and slide deck for Core Course C

### PR DESCRIPTION
**Pull request by terichadbourne**

### CODE REVIEW

File: CORE_AND_ELECTIVE_COURSES/CORE_COURSE_C/README.md [hunk 1]
```diff
@@ -31,3 +31,8 @@ Ways to use IPFS as a building block in your application:
   - Ensure `ipfs` command is available in your system, if possible
     - Suggested: install [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop#ipfs-desktop) (GUI+CLI for MacOS/Windows/Linux)
     - Alternative: [go-ipfs](https://docs.ipfs.io/introduction/usage/) (just CLI)
+
+## Course materials
+
+- Try the [demo](https://github.com/ipfs-shipyard/camp-course-c-demo)
+- View the [slide deck](https://docs.google.com/presentation/d/1cbJD5j_jRpm3yJiE6hLIOzMn8iMhvzyci7Ut3NLjqxs/edit?usp=sharing)
```
Comment (lidel) [inline CORE_AND_ELECTIVE_COURSES/CORE_COURSE_C/README.md]:
```suggestion
- View the slide deck: [PDF](https://github.com/ipfs/camp/files/3525750/IPFS.Camp.2019.-.Core.Course.C.-.Slide.Deck.pdf), [Google Docs](https://docs.google.com/presentation/d/1cbJD5j_jRpm3yJiE6hLIOzMn8iMhvzyci7Ut3NLjqxs/edit?usp=sharing)
```

File: CORE_AND_ELECTIVE_COURSES/CORE_COURSE_C/README.md [hunk 2]
```diff
@@ -31,3 +31,8 @@ Ways to use IPFS as a building block in your application:
   - Ensure `ipfs` command is available in your system, if possible
     - Suggested: install [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop#ipfs-desktop) (GUI+CLI for MacOS/Windows/Linux)
     - Alternative: [go-ipfs](https://docs.ipfs.io/introduction/usage/) (just CLI)
+
+## Course materials
+
+- Try the [demo](https://github.com/ipfs-shipyard/camp-course-c-demo)
```
Comment (lidel) [inline CORE_AND_ELECTIVE_COURSES/CORE_COURSE_C/README.md]:
```suggestion
- Try running the [demo app](https://github.com/ipfs-shipyard/camp-course-c-demo)
```

**Review by lidel — CHANGES_REQUESTED** [review]
I've created PDF version to remove reliance on proprietary Google Docs:
[IPFS.Camp.2019.-.Core.Course.C.-.Slide.Deck.pdf](https://github.com/ipfs/camp/files/3525750/IPFS.Camp.2019.-.Core.Course.C.-.Slide.Deck.pdf)

See also: https://github.com/ipfs-shipyard/camp-course-c-demo/pull/1

**Review by lidel — APPROVED** [review]

**Comment by terichadbourne** [comment]
@daviddias or someone with super admin powers, can you please merge this one?

**Comment by lidel** [comment]
@jessicaschilling  ?

**Pull request merged**